include/h2o.h

```diff
@@ -1563,6 +1584,10 @@ void h2o_process_request(h2o_req_t *req);
  * returns the first handler that will be used for the request
  */
 h2o_handler_t *h2o_get_first_handler(h2o_req_t *req);
+/**
+ * returns list of handlers and its size that will be used for the request
+ */
+size_t h2o_get_handlers(h2o_req_t *req, h2o_handler_t ***out);
```

Comment:
```suggestion
```
I think you removed the definition now that it is no longer used, but forgot to remove the declaration here?